ui/raidboss/data/06-ew/raid/p12s.ts

```diff
@@ -970,6 +1058,63 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'P12S Glaukopis First Cleave',
+      type: 'StartsUsing',
+      netRegex: { id: '82FC', source: 'Athena' },
+      response: (data, matches, output) => {
+        // don't use Responses.tankCleave(); we want to tell the non-targeted tank to swap
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          cleaveOnYou: Outputs.tankCleaveOnYou,
+          cleaveNoTarget: Outputs.tankCleave,
+          cleaveSwap: Outputs.tankSwap,
+          avoidTankCleave: Outputs.avoidTankCleave,
+        };
+        // Multiple players can be hit by the line cleave AoE,
+        // but we only care about the intended target for the Followup trigger
+        data.glaukopisFirstHit = matches.target;
+
+        if (data.me === matches.target)
+          return { alertText: output.cleaveOnYou!() };
+        if (data.role === 'tank')
+          return { alertText: output.cleaveSwap!() };
+        if (data.job === 'BLU')
+          return { alertText: output.cleaveNoTarget!() };
```

Comment:
Maybe healers should get a "tankbusters" call?

I'm not sure about the `BLU` call here either (not that this matters for a long time), but I'd put them either in the tank (you should swap) or healer (hey there's some tankbusters here) category.

Reply:
Sure, I think adding the healers to cleaveNoTarget makes sense.  For the BLU, I think I did this in the the way it's handled in Responses.tankCleave() - i.e., a generic alert that there is a tank cleave, without a specific instruction to swap or avoid (since we don't know about mimicry).  Right now, any BLU (not the initial target of the initial cleave) would receive cleaveNoTarget, which is where I'll also add the healers -- are you thinking something different?

Reply:
I wonder if it should be plural, since there's two? Like "tank cleaves" or something? What do you think?

Reply:
> I wonder if it should be plural, since there's two? Like "tank cleaves" or something? What do you think?

Hah, I was gritting my teeth about using the singular vs. plural too, but wasn't sure if this justified a permanent addition to Outputs.  But a little searching...how do you feel about borrowing these from rubicante-ex and adding to Outputs?

```
        tankBusterCleaves: {
          en: 'Tank Buster Cleaves',
          de: 'Tankbuster Cleaves',
          fr: 'Tankbuster cleaves',
          ja: 'MT・ST同時範囲攻撃',
          cn: '双T扇形死刑',
          ko: '광역 탱버',
        },
        avoidTankCleaves: {
          en: 'Avoid Tank Cleaves',
          de: 'Tankbuster Cleaves ausweichen',
          fr: 'Évitez les cleaves (tankbusters)',
          ja: '範囲攻撃を避けて',
          cn: '躲避双T扇形死刑',
          ko: '광역 탱버 피하기',
        },
```

Reply:
Yeah, sounds good to me.